src/common/domain/Journal.d.ts

```diff
@@ -0,0 +1,369 @@
+/* tslint:disable */
```

Comment:
should we not be using the common schema on npm here?

Reply:
I agree with Tom, we should bring the common schema from api-definitions

Reply:
Do you mean the `mes-journal-schema` ?

Reply:
yes